AdminUi/src/AdminUi/Program.cs

```diff
@@ -113,7 +115,9 @@ static void Configure(WebApplication app)
             .AddCustomHeader("X-Frame-Options", "Deny")
     );
 
-    if (app.Environment.IsLocal() || app.Environment.IsDevelopment())
+    var swaggerUiConfiguration = app.Configuration.GetSection("SwaggerUi");
+    var isSwaggerEnabled = swaggerUiConfiguration.GetValue<bool>("Enabled");
+    if (isSwaggerEnabled && (app.Environment.IsLocal() || app.Environment.IsDevelopment()))
```

Comment:
We don't need to check the environment anymore. That was kind of an ugly way of doing it.
```suggestion
    if (isSwaggerEnabled)
```